config/environments/test.rb

```diff
@@ -13,7 +13,8 @@
   config.eager_load = false
 
   # Configure static file server for tests with Cache-Control for performance.
-  config.serve_static_files   = false
+  # Changed to true by rake rails:update
+  config.serve_static_files   = true
```

Comment:
That's a good question! I THINK what this does is makes stuff in the public folder accessible, but I think it'll work fine for us either way.